src/components/events/event/EventPage.jsx

```diff
@@ -312,7 +312,18 @@ const EventPage = () => {
               <p className="header">Waivers</p>
               {/* TODO Multiple waiver downloads; currently, only single waiver download button */}
               {eventData.waivers && eventData.waivers.length > 0 && (
-                <a href={eventData.waivers[0].link} download={eventData.waivers[0].name}>
+                <a
+                  href={
+                    eventData.waivers.filter(waiver => {
+                      return waiver.userId !== null;
```

Comment:
you can condense lines 317-319 like this:
`....filter(waiver => waiver.userId !== null)[0].link`

If you do the return in one line, you don't need the return keyword + another set of curly braces